### DISCUSSION TITLE
I can‘t login my local dify!

### DISCUSSION THREAD

**Discussion by jeremygan2021** [langgenius/dify#10230] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

![微信图片_20241104142548](https://github.com/user-attachments/assets/7205d6ac-b07c-4bb2-bd5f-9e011f59d894)

My local server of dify right now is unable to access due to the '''Authentication method not configured

Please contact the system admin to add an authentication method.'''

### 2. Additional context or comments

I  add my account already， but right now issues that I can't access。I build this projects on docker-compose。

**Comment by jeremygan2021** [comment]
![image](https://github.com/user-attachments/assets/5bb733d1-7ba5-4b44-9f18-9d3c6e72f0fa)
when I add admit account, it shows this!

**Comment by fjhmaoer123** [comment]
Me too, how can we solve it?

**Comment by BryceWG** [comment]
Use the latest .env.example file, add your old settings to it and rename it to .env

**Reply by qq569513407** [reply]
useful

**Comment by sinopec** [comment]
me too

**Comment by yangxuan21** [comment]
Me too, how can we solve it?

**Comment by sudo-tahirsaeed** [comment]
anyone got solution?

**Comment by fravala** [comment]
Has no one found the solution to this problem?

**Comment by crazytiy** [comment]
Check whether the Docker containers for api, worker, and sandbox are running. If any of them fail to start, edit the docker-compose.yaml file. Under the services section, add privileged: true to the corresponding service, such as api. Do the same for other services that fail to start. This should allow them to start normally.

**Comment by prodkt** [comment]
Anyone??

**Comment by Arkharantika** [comment]
Try to delete all of your images/container which already created, then install the new one by creating new folder / project from "quick start" guide , (dont follow the local source code start -> because i think this version is not stable yet) this work for me

<img width="945" height="707" alt="Screenshot 2026-02-03 122727" src="https://github.com/user-attachments/assets/b8b0c7cc-5854-48d0-b2c0-c47a086d1760" />
